Jupyter_notebook/tutorial.ipynb

```diff
@@ -240,9 +252,9 @@
       },
       "outputs": [],
       "source": [
-        "files = glob.glob(os.path.join(path, 'CCF/*.h5'))\n",
+        "files = glob.glob(os.path.join(stack_data_path, '**/*.h5'))\n",
```

Comment:
I believe this was supposed to be `STACK` instead of `CCF`? I updated it but this still didn't work for me. The stack step only created a single file: `./STACK/CI.ARV/CI.ARV_CI.BAK.h5`

And there were lots of warnings:
```
continue! not enough cross components for auto-correlation CI.ARV_CI.ARV in [/Users/carlosg/tmp/CCF/2019_02_01_00_00_00T2019_02_01_04_00_00.h5](https://file+.vscode-resource.vscode-cdn.net/Users/carlosg/tmp/CCF/2019_02_01_00_00_00T2019_02_01_04_00_00.h5)
continue! not enough cross components for auto-correlation CI.ARV_CI.ARV in [/Users/carlosg/tmp/CCF/2019_02_01_04_00_00T2019_02_01_08_00_00.h5](https://file+.vscode-resource.vscode-cdn.net/Users/carlosg/tmp/CCF/2019_02_01_04_00_00T2019_02_01_08_00_00.h5)
continue! not enough cross components for auto-correlation CI.ARV_CI.ARV in [/Users/carlosg/tmp/CCF/2019_02_01_08_00_00T2019_02_01_12_00_00.h5](https://file+.vscode-resource.vscode-cdn.net/Users/carlosg/tmp/CCF/2019_02_01_08_00_00T2019_02_01_12_00_00.h5)
continue! not enough cross components for auto-correlation CI.ARV_CI.ARV in [/Users/carlosg/tmp/CCF/2019_02_01_12_00_00T2019_02_01_16_00_00.h5](https://file+.vscode-resource.vscode-cdn.net/Users/carlosg/tmp/CCF/2019_02_01_12_00_00T2019_02_01_16_00_00.h5)
continue! not enough cross components for auto-correlation CI.ARV_CI.ARV in [/Users/carlosg/tmp/CCF/2019_02_01_16_00_00T2019_02_01_20_00_00.h5](https://file+.vscode-resource.vscode-cdn.net/Users/carlosg/tmp/CCF/2019_02_01_16_00_00T2019_02_01_20_00_00.h5)
continue! not enough cross components for auto-correlation CI.ARV_CI.ARV in [/Users/carlosg/tmp/CCF/2019_02_01_20_00_00T2019_02_02_00_00_00.h5](https://file+.vscode-resource.vscode-cdn.net/Users/carlosg/tmp/CCF/2019_02_01_20_00_00T2019_02_02_00_00_00.h5)
continue! not enough cross components for auto-correlation CI.ARV_CI.ARV in [/Users/carlosg/tmp/CCF/2019_02_02_00_00_00T2019_02_02_04_00_00.h5](https://file+.vscode-resource.vscode-cdn.net/Users/carlosg/tmp/CCF/2019_02_02_00_00_00T2019_02_02_04_00_00.h5)
continue! not enough cross components for auto-correlation CI.ARV_CI.ARV in [/Users/carlosg/tmp/CCF/2019_02_02_04_00_00T2019_02_02_08_00_00.h5](https://file+.vscode-resource.vscode-cdn.net/Users/carlosg/tmp/CCF/2019_02_02_04_00_00T2019_02_02_08_00_00.h5)
continue! not enough cross components for auto-correlation CI.ARV_CI.ARV in [/Users/carlosg/tmp/CCF/2019_02_02_08_00_00T2019_02_02_12_00_00.h5](https://file+.vscode-resource.vscode-cdn.net/Users/carlosg/tmp/CCF/2019_02_02_08_00_00T2019_02_02_12_00_00.h5)
continue! not enough cross components for auto-correlation CI.ARV_CI.ARV in [/Users/carlosg/tmp/CCF/2019_02_02_12_00_00T2019_02_02_16_00_00.h5](https://file+.vscode-resource.vscode-cdn.net/Users/carlosg/tmp/CCF/2019_02_02_12_00_00T2019_02_02_16_00_00.h5)
continue! not enough cross components for auto-correlation CI.ARV_CI.ARV in [/Users/carlosg/tmp/CCF/2019_02_02_16_00_00T2019_02_02_20_00_00.h5](https://file+.vscode-resource.vscode-cdn.net/Users/carlosg/tmp/CCF/2019_02_02_16_00_00T2019_02_02_20_00_00.h5)
continue! not enough cross components for auto-correlation CI.ARV_CI.ARV in [/Users/carlosg/tmp/CCF/2019_02_02_20_00_00T2019_02_03_00_00_00.h5](https://file+.vscode-resource.vscode-cdn.net/Users/carlosg/tmp/CCF/2019_02_02_20_00_00T2019_02_03_00_00_00.h5)
continue! not enough cross components for auto-correlation CI.ARV_CI.ARV in [/Users/carlosg/tmp/CCF/2019_02_03_00_00_00T2019_02_03_04_00_00.h5](https://file+.vscode-resource.vscode-cdn.net/Users/carlosg/tmp/CCF/2019_02_03_00_00_00T2019_02_03_04_00_00.h5)
continue! not enough cross components for auto-correlation CI.ARV_CI.ARV in [/Users/carlosg/tmp/CCF/2019_02_03_04_00_00T2019_02_03_08_00_00.h5](https://file+.vscode-resource.vscode-cdn.net/Users/carlosg/tmp/CCF/2019_02_03_04_00_00T2019_02_03_08_00_00.h5)
continue! not enough cross components for auto-correlation CI.ARV_CI.ARV in [/Users/carlosg/tmp/CCF/2019_02_03_08_00_00T2019_02_03_12_00_00.h5](https://file+.vscode-resource.vscode-cdn.net/Users/carlosg/tmp/CCF/2019_02_03_08_00_00T2019_02_03_12_00_00.h5)
continue! not enough cross components for auto-correlation CI.ARV_CI.ARV in [/Users/carlosg/tmp/CCF/2019_02_03_12_00_00T2019_02_03_16_00_00.h5](https://file+.vscode-resource.vscode-cdn.net/Users/carlosg/tmp/CCF/2019_02_03_12_00_00T2019_02_03_16_00_00.h5)
continue! not enough cross components for auto-correlation CI.ARV_CI.ARV in [/Users/carlosg/tmp/CCF/2019_02_03_16_00_00T2019_02_03_20_00_00.h5](https://file+.vscode-resource.vscode-cdn.net/Users/carlosg/tmp/CCF/2019_02_03_16_00_00T2019_02_03_20_00_00.h5)
continue! not enough cross components for auto-correlation CI.ARV_CI.ARV in [/Users/carlosg/tmp/CCF/2019_02_03_20_00_00T2019_02_04_00_00_00.h5](https://file+.vscode-resource.vscode-cdn.net/Users/carlosg/tmp/CCF/2019_02_03_20_00_00T2019_02_04_00_00_00.h5)
continue! not enough cross components for auto-correlation CI.ARV_CI.ARV in [/Users/carlosg/tmp/CCF/2019_02_04_00_00_00T2019_02_04_04_00_00.h5](https://file+.vscode-resource.vscode-cdn.net/Users/carlosg/tmp/CCF/2019_02_04_00_00_00T2019_02_04_04_00_00.h5)
continue! not enough cross components for auto-correlation CI.ARV_CI.ARV in [/Users/carlosg/tmp/CCF/2019_02_04_04_00_00T2019_02_04_08_00_00.h5](https://file+.vscode-resource.vscode-cdn.net/Users/carlosg/tmp/CCF/2019_02_04_04_00_00T2019_02_04_08_00_00.h5)
continue! not enough cross components for auto-correlation CI.ARV_CI.ARV in [/Users/carlosg/tmp/CCF/2019_02_04_08_00_00T2019_02_04_12_00_00.h5](https://file+.vscode-resource.vscode-cdn.net/Users/carlosg/tmp/CCF/2019_02_04_08_00_00T2019_02_04_12_00_00.h5)
continue! not enough cross components for auto-correlation CI.ARV_CI.ARV in [/Users/carlosg/tmp/CCF/2019_02_04_12_00_00T2019_02_04_16_00_00.h5](https://file+.vscode-resource.vscode-cdn.net/Users/carlosg/tmp/CCF/2019_02_04_12_00_00T2019_02_04_16_00_00.h5)
continue! not enough cross components for auto-correlation CI.ARV_CI.ARV in [/Users/carlosg/tmp/CCF/2019_02_04_16_00_00T2019_02_04_20_00_00.h5](https://file+.vscode-resource.vscode-cdn.net/Users/carlosg/tmp/CCF/2019_02_04_16_00_00T2019_02_04_20_00_00.h5)
...
continue! no pair of CI.BAK_CI.BCW in [/Users/carlosg/tmp/CCF/2019_02_04_16_00_00T2019_02_04_20_00_00.h5](https://file+.vscode-resource.vscode-cdn.net/Users/carlosg/tmp/CCF/2019_02_04_16_00_00T2019_02_04_20_00_00.h5)
continue! no pair of CI.BAK_CI.BCW in [/Users/carlosg/tmp/CCF/2019_02_04_20_00_00T2019_02_05_00_00_00.h5](https://file+.vscode-resource.vscode-cdn.net/Users/carlosg/tmp/CCF/2019_02_04_20_00_00T2019_02_05_00_00_00.h5)
```